public/index.html

```diff
@@ -357,8 +369,7 @@
                     <ul>
                         <li>GC802 and GC1111 now correctly assigned to Hiteon ES.  Thank you Julie!</li>
 
-                    </ul>
-                    <!-- Archive
```

Comment:
This removes the closing </ul> but doesn't restore it anywhere. I guess it's in the Archive comment block.